README.md

```diff
@@ -1,43 +1,22 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Template for HTML and CSS projects
-
-> A template repo that can be used for html and css projects
-> The template includes tools to fix linter errors.
+## My Portfolio
 
+- Build a personal portfolio site
+- Used my template which is created using linter tools for HTML and CSS
+- Used Figma design to create a UI
+- Used Flexbox to place elements on the page
 
 ## Built With
 
 - html
 - css
+- Flexbox
 
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
```

Comment:
- [ OPTIONAL] I would recommend you host your site and put a live demo link to allow visitors to view your work visually.or if not yet hosted you can remove broken link from from readme to make it professional.